Ugly fire alarm resprite

comma opened the pull request:
![](https://i.imgur.com/CWfahaz.png)
Shitty codersprite to make it waay smaller and fit bit better on new walls.
Also automatically adjusts pixelshifts because hahaha not remapping every damn alarm.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

PsiOmegaDelta commented:
As mentioned in some other PR, I don't much like pixel-shifting depending on the relevant wall being a simulated one.

comma commented:
Oh yeah can axe that part, it was just a copypaste. I tried to move it into a proc but lightbulbs and these have different directions where 'wall' should be for different dirs (and fire alarms even have different logic depending on their direction  ree)

comma commented:
Done and now with torch alarm states too

comma committed:
Ugly fire alarm resprite

Shitty codersprite to make it waay smaller and fit bit better on new walls.
Also automatically adjusts pixelshifts because hahaha not remapping every damn alarm.
